frontend/src/Poppins/OFL.txt

```diff
@@ -0,0 +1,93 @@
+Copyright 2020 The Poppins Project Authors (https://github.com/itfoundry/Poppins)
```

Comment:
keep only the fonts we'll be using.

Reply:
yeah this font is used in your figma file

Reply:
as it was used in figma design i used this font please check

Reply:
ok changed font to Roboto